src/components/ListItem.js

```diff
@@ -55,26 +65,30 @@ const ListItem = ({ item, onDelete, token }) => {
   }
 
   return (
-    <li className={className}>
-      <span className="label" aria-hidden="true">
-        {purchaseNext}
-      </span>
-      {item.name}
-      <span className="screen-reader-only">
-        Next purchase in {item.next_purchase} days.
-      </span>
-      <button
-        className={isPurchased ? 'purchased' : 'not-purchased'}
-        onClick={onHandle}
-        disabled={isPurchased ? !null : null}
-      >
-        Purchase
-        <span className="screen-reader-only">{item.name}.</span>
-      </button>
-      <button onClick={onDelete}>
-        <img className="trash" src={trash} alt="delete item" />
-      </button>
-    </li>
+    <>
+      <li className={className}>
+        <span className="label" aria-hidden="true">
+          {purchaseNext}
+        </span>
+        {item.name}
+        <span className="screen-reader-only">
+          Next purchase in {item.next_purchase} days.
+        </span>
+        <button
+          className={isPurchased ? 'purchased' : 'not-purchased'}
+          onClick={onHandle}
+          disabled={isPurchased ? !null : null}
```

Comment:
Not from this PR, but this expression can be simply `disabled={!isPurchased}`. 
```suggestion
          disabled={!isPurchased}
```